comptest/web/models.py

```diff
@@ -127,3 +147,22 @@ class ContentFile(models.Model):
 
     def __str__(self):
         return self.title
+
+
+class SubmissionMetadata(SingletonModel):
```

Comment:
Given that we have a bunch of other settings that should probably be managed via the admin panel (everything with a `SITE` prefix in https://github.com/2i2c-org/unnamed-thingity-thing/blob/b941c59cdc383f7237a56722cb864c5a94842cfe/comptest/comptest/settings.py#L221), can we call this something like `SiteConfiguration` (based on the example in django-solo (https://github.com/lazybird/django-solo)) instead?

Reply:
This can be done after the fact, merging now to avoid merge conflicts!